tailwind.config.ts

```diff
@@ -13,6 +13,30 @@ const config: Config = {
         "gradient-conic":
           "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
       },
+      colors: {
```

Comment:
I was not able to figure out how to add:
Gradient 43B7FE -> 4F48E6,
Primary 900 42BBFF - 10%,
Purple 900 9542FF - 10%,
Green 900 42FF77 - 10%